spidy/cam_cap/capture.cpp

```diff
@@ -0,0 +1,29 @@
+#include <ros/ros.h>
+#include <image_transport/image_transport.h>
+#include <opencv2/highgui/highgui.hpp>
+#include <cv_bridge/cv_bridge.h>
+
+void imageCallback(const sensor_msgs::ImageConstPtr& msg)
+{
+  try
+  {
+    cv::imshow("view", cv_bridge::toCvShare(msg, "bgr8")->image);
```

Comment:
i think you do not need in that file

Reply:
i was using it at first and forget to remove it 
now i remove the whole file